rest-api/src/main/java/gov/cms/qpp/conversion/api/helper/MetadataHelper.java

```diff
@@ -33,16 +37,23 @@ public static Metadata generateMetadata(Node node, Outcome outcome) {
 
 		Metadata metadata = new Metadata();
 
-		metadata.setApm(findApm(node));
+		String apmId = findApm(node);
+
+		metadata.setApm(apmId);
```

Comment:
Why was this changed?

Reply:
Noise from when I was experimenting.  I'll change it back.